frontend/apps/cloudstorage/cloudstorage.lua

```diff
@@ -23,14 +23,6 @@ local Screen = require("device").screen
 local T = require("ffi/util").template
 
 local CloudStorage = Menu:extend{
-    cloud_servers = {
-        {
-            text = _("Add new cloud storage"),
-            title = _("Choose cloud type"),
-            url = "add",
-            editable = false,
-        },
-    },
```

Comment:
I'm a bit confused by this one? Isn't that where you choose WebDAV vs Dropbox etc.?